lib/tls.js

```diff
@@ -174,13 +171,10 @@ exports.checkServerIdentity = function checkServerIdentity(host, cert) {
 
     if (!valid) {
       if (cert.subjectaltname) {
-        reason = util.format('Host: %s is not in the cert\'s altnames: %s',
-                             host,
-                             cert.subjectaltname);
+        reason =
+          `Host: ${host} is not in the cert's altnames: ${cert.subjectaltname}`;
```

Comment:
Style nit: line continuations should indent by four spaces.  Didn't the linter complain?